interface/skin.dmf

```diff
@@ -1375,6 +1375,15 @@ menu "menu"
 		is-disabled = false
 		saved-params = "is-checked"
 	elem 
+		name = "Reconnect"

```

Comment:
Given the fact that every other element has its name set as `&<thing>`, this should probably be `&Reconnect`.